lib/Entities/SubscriberEntity.php

```diff
@@ -0,0 +1,350 @@
+<?php
+
+namespace MailPoet\Entities;
+
+use DateTimeInterface;
+use MailPoet\Doctrine\EntityTraits\AutoincrementedIdTrait;
+use MailPoet\Doctrine\EntityTraits\CreatedAtTrait;
+use MailPoet\Doctrine\EntityTraits\DeletedAtTrait;
+use MailPoet\Doctrine\EntityTraits\UpdatedAtTrait;
+use MailPoetVendor\Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="subscribers")
+ */
+class SubscriberEntity {
+  // statuses
+  const STATUS_BOUNCED = 'bounced';
+  const STATUS_INACTIVE = 'inactive';
+  const STATUS_SUBSCRIBED = 'subscribed';
+  const STATUS_UNCONFIRMED = 'unconfirmed';
+  const STATUS_UNSUBSCRIBED = 'unsubscribed';
+
+  use AutoincrementedIdTrait;
+  use CreatedAtTrait;
+  use UpdatedAtTrait;
+  use DeletedAtTrait;
+
+  /**
+   * @ORM\Column(type="bigint")
+   * @var int|null
+   */
+  private $wp_user_id;
+
+  /**
+   * @ORM\Column(type="boolean")
+   * @var bool|null
+   */
+  private $is_woocommerce_user;
```

Comment:
This is not nullable in DB and has default value 0 so I guess we could set this property as `false` by default and remove null from the annotation.